app/src/main/res/values/dimens.xml

```diff
@@ -7,4 +7,11 @@
     <dimen name="sp_30">30sp</dimen>
     <dimen name="dp_125">125dp</dimen>
     <dimen name="dp_150">150dp</dimen>
+    <dimen name="dp_100">100dp</dimen>
+    <dimen name="sp_15">15sp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.sp_15` appears to be unused
 Unused resources make applications larger and slow down builds.